packages/component-library-react/src/ButtonGroup.tsx

```diff
@@ -4,38 +4,5 @@
  * Copyright (c) 2021-2024 Gemeente Utrecht
  */
 
-import clsx from 'clsx';
-import { ForwardedRef, forwardRef, HTMLAttributes, isValidElement, PropsWithChildren, ReactNode } from 'react';
-
-const hasManyElements = (children: ReactNode | ReactNode[]) =>
-  Array.isArray(children) &&
-  children.reduce((count: number, item): number => (isValidElement(item) ? count + 1 : count), 0) >= 2;
-
-export interface ButtonGroupProps extends HTMLAttributes<HTMLParagraphElement> {
-  direction?: string | 'column' | 'row';
-}
-
-export const ButtonGroup = forwardRef(
-  (
-    { children, className, direction, ...restProps }: PropsWithChildren<ButtonGroupProps>,
-    ref: ForwardedRef<HTMLParagraphElement>,
-  ) => (
-    <p
-      role={hasManyElements(children) ? 'group' : undefined}
-      {...restProps}
-      ref={ref}
-      className={clsx(
-        'utrecht-button-group',
-        {
-          'utrecht-button-group--column': direction === 'column',
-          'utrecht-button-group--row': direction === 'row',
-        },
-        className,
-      )}
-    >
-      {children}
-    </p>
-  ),
-);
-
-ButtonGroup.displayName = 'ButtonGroup';
+export { type ButtonGroupProps } from '@utrecht/button-group-react';
```

Comment:
Dit is alleen nodig als je `ButtonGroupProps` ergens importeert. Prima om het zo te laten als het in andere packages ook zo gedaan is, dan moet het later een keer rechtgetrokken worden.

Reply:
Prima om later eens recht te trekken idd